frontend/src/components/SyncEditor/SyncEditor.test.tsx

```diff
@@ -292,6 +293,16 @@ describe('SyncEditor component', () => {
         getter: getValue.bind(null, 'Policy[0].spec.policy-templates[0].objectDefinition.spec.severity'),
         setState: setAwsAccessKeyID,
       },
+      {
+        path: `ClusterCurator[0].spec.install.jobMonitorTimeout}`,
+        setter: ((value: any) => {
+          const errors: any[] = []
+          const path = `ClusterCurator[0].spec.prehook`
+          errors.push({ path, message: 'Must be Job or Workflow' })
+          setInstallTimeout(value)
+          return errors
+        }).bind(null),
+      },
```

Comment:
This branch may be out of date... was the work started before we pulled the timeout story?